lib/chiscore/support/data_exporter.rb

```diff
@@ -0,0 +1,76 @@
+require 'csv'
+require 'pp'
+
+require 'chiscore/teams'
+require 'chiscore/checkins'
+require 'chiscore/checkpoints'
+require 'chiscore/routes'
+require 'chiscore/logins'
+require 'chiscore/repository'
+require 'chiscore/repository/redis_strategy'
+require 'chiscore/support/data_importer'
+
+ChiScore::Repository.set_strategy(ChiScore::RedisStrategy)
+ChiScore::DataImporter.import_for(2017)
+
+module ChiScore
+  class DataExporter
+
+    def export
+      results = ChiScore::Teams.all.map do |team|
+        values_for(team)
+      end.compact
+
+      csv_str = CSV.generate do |csv|
+        results.each { |row| csv << row }
+      end
+
+      puts
+      puts
+      puts "Delete anything above this line (including this line)"
```

Comment:
I can't output to the filesystem on Heroku, because of how it handles files, so I'm instead writing this out to stdout

The idea being that you can, say

```
heroku run "rake export" > some-output-file.csv
```

The problem is that Heroku adds a bunch of output around running a remote task, which munges the CSV. By adding this line, we can just delete that crap manually and get a valid CSV file.